example-config/gremlin-server.yaml

```diff
@@ -0,0 +1,39 @@
+host: 0.0.0.0
+port: 8182
+threadPoolWorker: 1
+gremlinPool: 8
+scriptEvaluationTimeout: 30000
+serializedResponseTimeout: 30000
+channelizer: org.apache.tinkerpop.gremlin.server.channel.WebSocketChannelizer
+graphs: { graph: /etc/opt/janusgraph/janusgraph.properties }
+
```

Comment:
I think the `example-config` directory could become problematic as the config is version specific. This config for example won't work with JanusGraph 0.3.0 as the way `gremlin-groovy` has to be configured changed in TinkerPop 3.3.0.
Also this file seems to be an exact copy of that in the `0.2.0` directory which means that we always have to make changes in both places.

Couldn't we simply use the config files from the version specific directories instead and remove this `example-config` directory?